linux/base.Dockerfile

```diff
@@ -215,4 +215,16 @@ RUN curl -fsSL https://aka.ms/install-azd.sh | bash && \
   # Add soft links
   #
   ln -s /usr/bin/python3 /usr/bin/python && \
-  ln -s /usr/bin/node /usr/bin/nodejs
+  ln -s /usr/bin/node /usr/bin/nodejs && \
+  #
+  # Install rootless kit
+  TMP_DIR=$(mktemp -d) && \
+  pushd $TMP_DIR && \
+  ROOTLESSKIT_VERSION=$(curl https://api.github.com/repos/rootless-containers/rootlesskit/releases/latest | jq -r '.tag_name') && \
```

Comment:
We can update, without having the intention, to a new minor or even major like this.

For example, release 2.3.0 deprecated docker-proxy, that IIUC is what we are using (maybe it is the rootless version and it is different?), and in some release in the future it might be removed and our setup breaks. It will be even hard to bisect due to how we handle it here.

My personal preference here is to put a fixed release and update on purpose ourselves (and often), that also enables bisect to work fine in case we need it for some hairy issue.

Reply:
@rata it is on purpose to have all the latest stuff in here, otherwise it is a maintenance nightmare to update each and every tool shipped as a part of this docker file. Hence none of other tools are pinned.

Reply:
Okay, thanks! Feel free to close this (it seems I can't)